### PR TITLE
Added job deadline information

### PR DESCRIPTION
**Pull request by soltysh**
This needs to wait for the next rebase. I'll remove the do_not_merge tag when appropriate.

In the mean time @bfallonf ptal

### CODE REVIEW

**Comment by bfallonf** [comment]
@soltysh Looks ok to me. Let me know when it's ok to go and I can merge it. (Might do some follow-up edits, too.)

File: dev_guide/jobs.adoc [hunk 1]
```diff
@@ -92,3 +92,29 @@ $ oc scale job pi --replicas=3
...
+The following example shows the part of a `*Job*` specifying
+`*activeDeadlineSeconds*` field for 30 minutes:
+
+====
+----
+{
+  "spec" : {
+    "activeDeadlineSeconds" : 1800,
+  }
+}
```
Comment (adellape) [inline dev_guide/jobs.adoc]: This can be YAML to match the rest of the topic.

**Comment by adellape** [comment]
@soltysh I take it this has made it into origin by now; can you confirm?

**Comment by soltysh** [comment]
@adellape not yet, it needs to wait for the next k8s rebase. It didn't make into the current one :(

**Comment by soltysh** [comment]
@adellape although one rebase landed (see https://github.com/openshift/origin/pull/6320) ADS didn't made it yet. Need to wait for another with this one.

**Comment by soltysh** [comment]
@adellape with last rebase in (https://github.com/openshift/origin/pull/6947), this is good to go.

**Pull request merged**